docs/devconsole/setup-gitops.md

```diff
@@ -0,0 +1,89 @@
+## Steps to follow for seting up a sample GitOps pipelines using [kam](../../releases)
+
+- Complete all the pre-requisite steps mentioned in the [docs](/docs/journey/day1#day-1-operations)
+- Bootstrap a GitOps repository following steps mentioned in the [docs](/docs/journey/day1#bootstrapping-the-manifest)
+
+  ```sh
+  $ kam bootstrap --service-repo-url https://github.com/<username>/taxi.git \          
+    --dockercfgjson ~/Downloads/<username>-gitops-auth.json \
+    --gitops-repo-url https://github.com/<username>/gitops-example.git \
+    --image-repo quay.io/<username>/taxi --prefix demo --output <output directory> \
+    --sealed-secrets-svc sealed-secrets-controller --sealed-secrets-ns kube-system
+
+  ```
+
+- This bootstraps your GitOps repo with two environments `demo-dev` and `demo-stage`.
+  - `demo-dev` has an app `app-taxi` with one service `taxi`.
+- Update the labels of the deployment of taxi service.
+  - Copy labels from `environments/demo-dev/apps/app-taxi/services/taxi/base/config/200-service.yaml` to `environments/demo-dev/apps/app-taxi/services/taxi/base/config/100-deployment.yaml` 
+- Add a new service `bus` to the created app `app-taxi` in `demo-dev` environment - 
+
+  ```sh
+  $ kam service add  --app-name app-taxi --env-name demo-dev \
+    --pipelines-folder <bootstrapped gitops folder> \
+    --service-name bus --sealed-secrets-ns kube-system \
+    --sealed-secrets-svc sealed-secrets-controller \
+    --git-repo-url https://github.com/<username>/bus.git
+  ```
+
+- Copy `config` folder from `taxi/base/` to `bus/base/`.
+  - Update `100-deployment.yaml` and `200-service.yaml` to change names and labels from `taxi` to `bus`.
+
+- Add `taxi` and `bus` service to `app-taxi` for `demo-stage` environment.
+
+  ```sh
+  $ kam service add  --app-name app-taxi --env-name demo-stage \
+    --pipelines-folder <bootstrapped gitops folder> \
+    --service-name taxi --sealed-secrets-ns kube-system \
+    --sealed-secrets-svc sealed-secrets-controller
```

Comment:
We don't have to pass sealed secrets info? The docs recommend creating the operator in `cicd` namespace

Reply:
Removed